client/server/middleware/logger.js

```diff
@@ -1,43 +1,57 @@
+/**
+ * External dependencies
+ */
+import uaParser from 'ua-parser-js';
+
 /**
  * Internal dependencies
  */
 import { getLogger } from 'server/lib/logger';
+import config from 'config';
 
 const NS_TO_MS = 1e-6;
 
+const parseUA = ( rawUA ) => {
+	const parsedUA = uaParser( rawUA );
+	if ( parsedUA.browser.name && parsedUA.browser.version ) {
+		const version = parsedUA.browser.version.split( '.' )[ 0 ];
+		return `${ parsedUA.browser.name } ${ version }`;
+	}
+	return rawUA;
+};
+
 const logRequest = ( req, res, options ) => {
-	const { requestStart } = options;
+	const { requestStart, env, version } = options;
 
-	let fields = {
+	const fields = {
 		method: req.method,
 		status: res.statusCode,
 		length: res.get( 'content-length' ),
 		url: req.originalUrl,
 		duration: Number(
 			( Number( process.hrtime.bigint() - requestStart ) * NS_TO_MS ).toFixed( 3 )
 		),
+		httpVersion: req.httpVersion,
+		version,
```

Comment:
I have second thoughts about the naming of this one. "Version" seems too vague in my mind - is it the browser version? the node version? the HTTP version? the latest tagged version (I know we don't maintain those, but someone could possibly lack that knowledge). 

How do you feel about renaming this to `sha` or `commitSha` so it's more explicit what this version is?

Reply:
Follow up in #46200